--- Makefile.orig	Sun Dec 11 19:12:49 2022
+++ Makefile	Sun Dec 11 19:13:49 2022
@@ -1,6 +1,6 @@
 COMMENT =	fast image processing library with low memory needs
 
-VERSION =	8.9.2
+VERSION =	8.13.0
 DISTNAME =	vips-${VERSION}
 PKGNAME =	lib${DISTNAME}
 
--- distinfo.orig	Sun Dec 11 19:12:59 2022
+++ distinfo	Sun Dec 11 19:14:31 2022
@@ -1,2 +1,2 @@
-SHA256 (vips-8.9.2.tar.gz) = roSRsRVs0uucu6ov1sqh3J7T3tC3BEPSjNf+p5irKic=
-SIZE (vips-8.9.2.tar.gz) = 17238831
+SHA256 (vips-8.13.0.tar.gz) = t+HVDc9XEWW+7NNq3s5uymcBwqnhMcZ1FD2LlBjb3YE=
+SIZE (vips-8.13.0.tar.gz) = 17892132
--- pkg/PLIST.orig	Sun Dec 11 19:13:20 2022
+++ pkg/PLIST	Sun Dec 11 19:30:08 2022
@@ -4,7 +4,7 @@
 bin/light_correct
 bin/shrink_width
 @bin bin/vips
-bin/vips-8.9
+bin/vips-8.13
 @bin bin/vipsedit
 @bin bin/vipsheader
 bin/vipsprofile
@@ -14,11 +14,11 @@
 include/vips/VError8.h
 include/vips/VImage8.h
 include/vips/VInterpolate8.h
+include/vips/VRegion8.h
 include/vips/almostdeprecated.h
 include/vips/arithmetic.h
 include/vips/basic.h
 include/vips/buf.h
-include/vips/cimg_funcs.h
 include/vips/colour.h
 include/vips/connection.h
 include/vips/conversion.h
@@ -53,7 +53,6 @@
 include/vips/resample.h
 include/vips/sbuf.h
 include/vips/semaphore.h
-include/vips/soname.h
 include/vips/thread.h
 include/vips/threadpool.h
 include/vips/transform.h
@@ -62,7 +61,6 @@
 include/vips/vector.h
 include/vips/version.h
 include/vips/video.h
-include/vips/vips-operators.h
 include/vips/vips.h
 include/vips/vips7compat.h
 include/vips/vips8
@@ -86,16 +84,16 @@
 share/gtk-doc/
 share/gtk-doc/html/
 share/gtk-doc/html/libvips/
-share/gtk-doc/html/libvips/Cite.md.html
+share/gtk-doc/html/libvips/Cite.html
 share/gtk-doc/html/libvips/Combine.png
-share/gtk-doc/html/libvips/Examples.md.html
-share/gtk-doc/html/libvips/How-it-opens-files.md.html
-share/gtk-doc/html/libvips/How-it-works.md.html
-share/gtk-doc/html/libvips/Making-image-pyramids.md.html
+share/gtk-doc/html/libvips/Examples.html
+share/gtk-doc/html/libvips/How-it-opens-files.html
+share/gtk-doc/html/libvips/How-it-works.html
+share/gtk-doc/html/libvips/Making-image-pyramids.html
 share/gtk-doc/html/libvips/Memtrace.png
 share/gtk-doc/html/libvips/Sequence.png
 share/gtk-doc/html/libvips/Sink.png
-share/gtk-doc/html/libvips/Using-vipsthumbnail.md.html
+share/gtk-doc/html/libvips/Using-vipsthumbnail.html
 share/gtk-doc/html/libvips/Vips-smp.png
 share/gtk-doc/html/libvips/VipsForeignSave.html
 share/gtk-doc/html/libvips/VipsImage.html
@@ -108,23 +106,14 @@
 share/gtk-doc/html/libvips/VipsThreadState.html
 share/gtk-doc/html/libvips/annotation-glossary.html
 share/gtk-doc/html/libvips/api-index-full.html
-share/gtk-doc/html/libvips/binding.md.html
+share/gtk-doc/html/libvips/binding.html
 share/gtk-doc/html/libvips/ch01.html
 share/gtk-doc/html/libvips/ch02.html
 share/gtk-doc/html/libvips/ch03.html
 share/gtk-doc/html/libvips/ch04.html
-share/gtk-doc/html/libvips/colour-management.html
-share/gtk-doc/html/libvips/compiled-language-which-can-call-c.html
-share/gtk-doc/html/libvips/cropping.html
-share/gtk-doc/html/libvips/documentation.html
-share/gtk-doc/html/libvips/dynamic-langauge-without-ffi.html
-share/gtk-doc/html/libvips/dynamic-language-with-ffi.html
 share/gtk-doc/html/libvips/extending.html
 share/gtk-doc/html/libvips/file-format.html
-share/gtk-doc/html/libvips/final-suggestion.html
-share/gtk-doc/html/libvips/full-decompression.html
 share/gtk-doc/html/libvips/func-list.html
-share/gtk-doc/html/libvips/gobject-introspection.html
 share/gtk-doc/html/libvips/home.png
 share/gtk-doc/html/libvips/index.html
 share/gtk-doc/html/libvips/interconvert.png
@@ -140,6 +129,7 @@
 share/gtk-doc/html/libvips/libvips-draw.html
 share/gtk-doc/html/libvips/libvips-error.html
 share/gtk-doc/html/libvips/libvips-freqfilt.html
+share/gtk-doc/html/libvips/libvips-from-C++.html
 share/gtk-doc/html/libvips/libvips-generate.html
 share/gtk-doc/html/libvips/libvips-header.html
 share/gtk-doc/html/libvips/libvips-histogram.html
@@ -155,22 +145,14 @@
 share/gtk-doc/html/libvips/libvips-version.html
 share/gtk-doc/html/libvips/libvips-vips.html
 share/gtk-doc/html/libvips/libvips.devhelp2
-share/gtk-doc/html/libvips/linear-light.html
-share/gtk-doc/html/libvips/looping.html
 share/gtk-doc/html/libvips/object-tree.html
-share/gtk-doc/html/libvips/output-directory.html
-share/gtk-doc/html/libvips/output-format-and-options.html
 share/gtk-doc/html/libvips/owl.jpg
-share/gtk-doc/html/libvips/random-access-via-load-library.html
 share/gtk-doc/html/libvips/right-insensitive.png
 share/gtk-doc/html/libvips/right.png
-share/gtk-doc/html/libvips/sequential-access.html
 share/gtk-doc/html/libvips/style.css
-share/gtk-doc/html/libvips/thumbnail-size.html
 share/gtk-doc/html/libvips/tn_owl.jpg
 share/gtk-doc/html/libvips/up-insensitive.png
 share/gtk-doc/html/libvips/up.png
 share/gtk-doc/html/libvips/using-cli.html
 share/gtk-doc/html/libvips/using-from-c.html
-share/gtk-doc/html/libvips/using-from-cpp.html
 share/gtk-doc/html/libvips/using-threads.html
--- patches/patch-libvips_include_vips_dispatch_h.orig	Fri Mar 11 13:22:46 2022
+++ patches/patch-libvips_include_vips_dispatch_h	Sun Dec 11 19:38:49 2022
@@ -1,22 +0,0 @@
-From cfe50903dffd576f6c25b0f24882b38a38085644 Mon Sep 17 00:00:00 2001
-From: John Cupitt <jcupitt@gmail.com>
-Date: Tue, 9 Feb 2021 16:07:51 +0000
-Subject: [PATCH] don't put glib includes inside extern "C" blocks
-
-Index: libvips/include/vips/dispatch.h
---- libvips/include/vips/dispatch.h.orig
-+++ libvips/include/vips/dispatch.h
-@@ -33,11 +33,12 @@
- #ifndef IM_DISPATCH_H
- #define IM_DISPATCH_H
- 
-+#include <glib-object.h>
-+
- #ifdef __cplusplus
- extern "C" {
- #endif /*__cplusplus*/
- 
--#include <glib-object.h>
- #include <vips/vips.h>
- #include <vips/util.h>
- 
--- patches/patch-libvips_include_vips_vips_h.orig	Fri Mar 11 13:22:46 2022
+++ patches/patch-libvips_include_vips_vips_h	Sun Dec 11 19:38:55 2022
@@ -1,27 +0,0 @@
-From cfe50903dffd576f6c25b0f24882b38a38085644 Mon Sep 17 00:00:00 2001
-From: John Cupitt <jcupitt@gmail.com>
-Date: Tue, 9 Feb 2021 16:07:51 +0000
-Subject: [PATCH] don't put glib includes inside extern "C" blocks
-
-Index: libvips/include/vips/vips.h
---- libvips/include/vips/vips.h.orig
-+++ libvips/include/vips/vips.h
-@@ -80,14 +80,14 @@
- #ifndef VIPS_VIPS_H
- #define VIPS_VIPS_H
- 
--#ifdef __cplusplus
--extern "C" {
--#endif /*__cplusplus*/
--
- #include <glib.h>
- #include <glib/gstdio.h>
- #include <gmodule.h>
- #include <glib-object.h>
-+
-+#ifdef __cplusplus
-+extern "C" {
-+#endif /*__cplusplus*/
- 
- /* If we're being parsed by SWIG, remove gcc attributes.
-  */
